### ISSUE TITLE
Problem with fetch_balance on Bybit future market

### ISSUE THREAD

**Issue by seeyoul8r** [ccxt/ccxt#22909]
### Operating System

windows 11

### Programming Languages

Python

### CCXT Version

last

### Description

Hello! I'm trying to make logic for both markets futures and spot, Bybit exchange has one Unified Trading Account for both of this type of trades. I'm able to fetch my balance if i set defaultType == spot, but when i choose future i receive this blank fields. How to handle that issue? (attachment
![Screenshot 2024-06-27 114120](https://github.com/ccxt/ccxt/assets/124713131/5b3ddaf5-f0b4-4c1c-b720-9a79710b12fa)
)

### Code

```
  

```

**Comment by seeyoul8r** [comment]
I solved the problem, when i said future market type i meant perpetual swap.

**Comment by carlosmiei** [comment]
Hello @seeyoul8r, so is the issue resolved now?

**Comment by seeyoul8r** [comment]
> Hello @seeyoul8r, so is the issue resolved now?

Yes, thanks! I just need to use defaultType as swap instead future.

**Comment by carlosmiei** [comment]
@seeyoul8r Alternatively you can provide `type:swap` in `params` upon calling `fetch_balance`

**Comment by carlotestor** [comment]
This can be closed without further user input.

Reporter confirmed the issue is resolved (Bybit perpetual/swap balance vs “future” naming):

> Yes, thanks! I just need to use defaultType as swap instead future.

Alternatively: pass `params={'type': 'swap'}` to `fetch_balance`.
